### PR TITLE
⬆️  upgrades urllib3==1.26.5 to fix CVE-2021-33503

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

upgrades urllib3==1.26.5 to fix CVE-2021-33503 . This fixes vulnerability in ``director`` services as reported in
https://github.com/ITISFoundation/osparc-simcore/security/dependabot/97

### CODE REVIEW

File: scripts/common.Makefile [hunk 1]
```diff
@@ -165,7 +165,7 @@ pip-freeze: ## dumps current environ and base.txt [diagnostics]
 
 .PHONY: _check_python_version _check_venv_active
 
-_check_python_version:
+_check_python_versio%:
```
Comment (GitHK) [inline scripts/common.Makefile:168]: what is this `%` doing here?

File: services/director/requirements/_base.in [hunk 2]
```diff
@@ -57,14 +57,14 @@ prometheus-client==0.8.0
 psycopg2-binary==2.8.6
 pydantic==1.7.2
 PyYAML==5.4 # CVE-2020-1747
-requests==2.22.0
+requests
```
Comment (GitHK) [inline services/director/requirements/_base.in:60]: Could you pin back the library at version `requests==2.27.1` as seen below?

File: scripts/common.Makefile (hunk 1 above)
Comment (pcrespov) [reply scripts/common.Makefile:168]: This allows to override that recipe in the director without a warning. This is to makefile what * is for glob

File: services/director/requirements/_base.in (hunk 2 above)
Comment (pcrespov) [reply services/director/requirements/_base.in:60]: i forgot that. thx

File: services/director/requirements/_base.in [hunk 3]
```diff
@@ -57,14 +57,14 @@ prometheus-client==0.8.0
 psycopg2-binary==2.8.6
 pydantic==1.7.2
 PyYAML==5.4 # CVE-2020-1747
-requests==2.22.0
+requests==2.27.1 #
```
Comment (GitHK) [inline services/director/requirements/_base.in:60]: don't know if you intended to put a comment here

**Review by GitHK — APPROVED** [review]
👍

**Review by mrnicegyu11 — APPROVED** [review]

**Comment by mrnicegyu11** [comment]
Thanks a lot!

**Review by sanderegg — APPROVED** [review]

**Pull request merged**